modules/contact/test/tests/mortar_dynamics/tests

```diff
@@ -16,6 +16,19 @@
     requirement = 'The system shall solve mortar frictionless contact between two blocks with '
                   'weighted gap time stabilization using mortar nodal geometry.'
   []
+  [block-dynamics-reference]
+    type = 'CSVDiff'
+    input = 'block-dynamics-reference.i'
+    csvdiff = 'block-dynamics-reference_out.csv'
+    rel_err = 5.0e-4
+    abs_zero = 1.0e-5
+    min_parallel = 1
+    max_parallel = 4
```

Comment:
At some point I thought it may be useful to control the number of procs these tests can be run on. However, we don't do sweeps in modules. So they are pointless. I have removed them from that `tests` file.